### PR TITLE
Added get dogs route!

### LINKED ISSUE #19: Add get all dogs route

### PR DESCRIPTION
**Pull request by jonfinger**
Adds a get dog route. It also removes some random junk from models.py that lambda had me put in 🤷🏻

### CODE REVIEW

**Review by allie-rae — APPROVED** [review]
awwwww yeahhhhhhhhh

File: sniffr/dog_routes/dog_routes.py [hunk 1]
```diff
@@ -25,6 +28,47 @@ def get_dog(dog_id):
...
+                }
+            
+            response.append(dog)
+
+        return jsonify(response)
+
+    else:
+        response = {"message": "Dog Not Found"}
+        return response, 400
+
```
Comment (DeusExMashima) [inline sniffr/dog_routes/dog_routes.py:67]:
```suggestion
```

File: sniffr/dog_routes/dog_routes.py [hunk 2]
```diff
@@ -25,6 +28,47 @@ def get_dog(dog_id):
...
+# Get Dogs
+#
+
+@dog_bp.route("/dogs", methods=["GET"])
+@cross_origin()
+def get_dogs():
+
+    queried_dogs = db.session.query(Dog).join(User, Dog.owner_id==User.user_id).all()
+    response = []
+    if queried_dogs:
```
Comment (DeusExMashima) [inline sniffr/dog_routes/dog_routes.py:41]: Would ``map()`` make it look cuter?
```
https://realpython.com/python-map-function/
```

**Review by danimal-johnson — APPROVED** [review]
Looks good.

File: sniffr/dog_routes/dog_routes.py (hunk 2 above)
Comment (jonfinger) [reply sniffr/dog_routes/dog_routes.py:41]: I don't think so 😝

**Pull request merged**